packages/central-server/app/sync/updateLookupTable.js

```diff
@@ -132,17 +132,23 @@ export const updateLookupTable = withConfig(
     const sessionConfig = {};
 
     let changesCount = 0;
+    const modelsToRebuild = await models.LocalSystemFact.getLookupModelsToRebuild();
 
     for (const model of Object.values(outgoingModels)) {
       try {
+        const shouldFullyRebuild = modelsToRebuild.includes(model.tableName);
         const modelChangesCount = await updateLookupTableForModel(
           model,
           config,
-          since,
+          shouldFullyRebuild ? 0 : since,
```

Comment:
I'm pretty sure 0 is the right value here? Feels a little magic-numbery tho so might define a constant for it.